packages/desktop-client/src/components/modals/CreateAccountModal.tsx

```diff
@@ -364,9 +467,76 @@ export function CreateAccountModal({ upgradingAccountId }: CreateAccountProps) {
+                          <View
+                            style={{
+                              flexDirection: 'row',
+                              gap: 10,
+                              alignItems: 'center',
+                            }}
+                          >
+                            <ButtonWithLoading
+                              isDisabled={syncServerStatus !== 'online'}
+                              style={{
+                                padding: '10px 0',
+                                fontSize: 15,
+                                fontWeight: 600,
+                                flex: 1,
+                              }}
+                              onPress={onConnectPluggyAi}
+                            >
+                              {isPluggyAiSetupComplete
+                                ? t('Link bank account with Pluggy.ai')
+                                : t('Set up Pluggy.ai for bank sync')}
+                            </ButtonWithLoading>
+                            {isPluggyAiSetupComplete && (
+                              <DialogTrigger>
+                                <Button
+                                  variant="bare"
+                                  aria-label={t('Pluggy.ai menu')}
+                                >
+                                  <SvgDotsHorizontalTriple
+                                    width={15}
+                                    height={15}
+                                    style={{ transform: 'rotateZ(90deg)' }}
+                                  />
+                                </Button>
+
+                                <Popover>
+                                  <Menu
+                                    onMenuSelect={item => {
+                                      if (item === 'reconfigure') {
+                                        onPluggyAiReset();
+                                      }
+                                    }}
+                                    items={[
+                                      {
+                                        name: 'reconfigure',
+                                        text: t('Reset Pluggy.ai credentials'),
+                                      },
+                                    ]}
+                                  />
+                                </Popover>
+                              </DialogTrigger>
+                            )}
+                          </View>
+                          <Text style={{ lineHeight: '1.4em', fontSize: 15 }}>
+                            <strong>
+                              {t('Link a')} <em>{t('Brazilian')}</em>{' '}
+                              {t('bank account')}
+                            </strong>{' '}
+                            {t(
+                              'to automatically download transactions. Pluggy.ai provides reliable, up-to-date information from hundreds of banks.',
+                            )}
```

Comment:
This is difficult to translate as it'll generate 4 separate strings with no context. We prefer using `<Trans>` components where possible and wrapping the entire translatable string. This, for example, would be:

```suggestion
                            <Trans>
                              <strong>
                                Link a <em>Brazilian</em> bank account
                              </strong>{' '}
                              to automatically download transactions.
                              Pluggy.ai provides reliable, up-to-date information from hundreds of banks.
                            </Trans>
```